README.md

```diff
@@ -8,16 +8,28 @@
 
 ## Table of Contents
 
-1. [Overview](#overview)
-1. [Module Description - What the module does and why it is useful](#module-description)
-1. [Setup - The basics of getting started with chrony](#setup)
-   - [What chrony affects](#what-chrony-affects)
-   - [Setup requirements](#setup-requirements)
-   - [Beginning with chrony](#beginning-with-chrony)
-1. [Usage - Configuration options and additional functionality](#usage)
-1. [Reference - An under-the-hood peek at what the module is doing and how](#reference)
-1. [Limitations - OS compatibility, etc.](#limitations)
-1. [Copyright and License](#copyright-and-license)
+- [puppet-chrony](#puppet-chrony)
```

Comment:
Not sure we want to have to maintain a three-level table of contents. This change should at least be a separate commit though, since it's not exactly part of adding srv record support.